spec/lib/dor/release/item_spec.rb

```diff
@@ -32,22 +28,6 @@
     end
   end
 
-  it 'calls dor-fetcher-client to get the members, but only once' do
-    expect(@item.fetcher).to receive(:get_collection).once
-    while @n < 3
-      expect(@item.members).to eq @response
-      @n += 1
-    end
-  end
-
-  it 'gets the right value for item_members' do
-    expect(@item.item_members).to eq @response['items']
-  end
-
-  it 'gets the right value for sub_collections' do
-    expect(@item.sub_collections).to eq @response['sets'] + @response['collections']
-  end
-
```

Comment:
it looks like these specs didn't migrate anywhere ... would it make sense to have this be the content of a spec for the new MemberService class?

Reply:
Right, because we have other tests that provide 100% coverage of MemberService: https://coveralls.io/builds/28005421/source?filename=lib/dor/release/member_service.rb